tests/test_main_system_caches.py

```diff
@@ -131,19 +137,6 @@ def test_system_caches_delete_admin(self):
 
         # TODO: Unsure how to test log messages received from Connect.
 
-    # Admins cannot delete caches that do not exist
```

Comment:
This test should not be here imho,
it's testing a connect behaviour not `rsconnect-python`.

We are already testing that deleting a cache does work, and we are already testing that reporting an error message from delete works:

- `test_system_caches_delete_admin`
- `test_system_caches_delete_publisher`

So this test didn't test anything additional and make the testsuite too dependant on a connect behaviours that didn't break compatibility with rsconnect itself.